### DISCUSSION TITLE
Support for Utrecht Ms. 406

### DISCUSSION THREAD

**Discussion by dchiller** [DDMAL/cantus#705] [General]
Moving discussion here from #319.

To recap, there are two sets of available images for this manuscript. One have an accessible IIIF manifest, but are behind an authentication wall on DIAMM. The others are from an IIP (https://iipimage.sourceforge.io/) server at the host institution. 

We can discuss here the pros and cons of trying to get images from either of those sources on Cantus Ultimus.

**Comment by dchiller** [comment]
Theoretically, one should be able to make an IIIF request to an IIP server: https://iipimage.sourceforge.io/documentation/iiif

I'm not initially able to do this in this specific case, but this may be worth some additional poking around. Per Andrew's question: we would also need to verify that what we're doing doesn't violate any terms.

**Reply by ahankinson** [reply]
The question about DIAMM access is more about having someone available to help me write and test the authentication method. We have agreement about the use, I just need some help re-working the [DIAMM code](https://github.com/diamm/diamm/) to put project-based auth in.

**Reply by fujinaga** [reply]
When you're here, you can recruit someone to help you.
